thingystuff.txt

```diff
@@ -0,0 +1 @@
query-data","pages":[{"title":"Browse Data","path":"/docs/query-your-data/explore-query-data/browse-data","pages":[{"title":"Introduction Data Explorer","path":"/docs/query-your-data/explore-query-data/browse-data/introduction-data-explorer"}]},{"title":"Dashboards","path":"/docs/query-your-data/explore-query-data/dashboards","pages":[{"title":"Add Custom Visualizations Your Dashboards","path":"/docs/query-your-data/explore-query-data/dashboards/add-custom-visualizations-your-dashboards"},{"title":"Dashboards Api","path":"/docs/query-your-data/explore-query-data/dashboards/dashboards-api"},{"title":"Dashboards Charts Import Export Data","path":"/docs/query-your-data/explore-query-data/dashboards/dashboards-charts-import-export-data"},{"title":"Explore Public Api Performance Dashboard","path":"/docs/query-your-data/explore-query-data/dashboards/explore-public-api-performance-dashboard"},{"title":"Filter New Relic One Dashboards Facets","path":"/docs/query-your-data/explore-query-data/dashboards/filter-new-relic-one-dashboards-facets"},{"title":"Introduction Dashboards","path":"/docs/query-your-data/explore-query-data/dashboards/introduction-dashboards"},{"title":"Manage Your Dashboard","path":"/docs/query-your-data/explore-query-data/dashboards/manage-your-dashboard"},{"title":"New Relic Global Performance Data Sets","path":"/docs/query-your-data/explore-query-data/dashboards/new-relic-global-performance-data-sets"}]},{"title":"Get Started","path":"/docs/query-your-data/explore-query-data/get-started","pages":[{"title":"Introduction Querying New Relic Data","path":"/docs/query-your-data/explore-query-data/get-started/introduction-querying-new-relic-data"}]},{"title":"Query Builder","path":"/docs/query-your-data/explore-query-data/query-builder","pages":[{"title":"Introduction Query Builder","path":"/docs/query-your-data/explore-query-data/query-builder/introduction-query-builder"},{"title":"Nrql Console","path":"/docs/query-your-data/explore-query-data/query-builder/nrql-console"},{"title":"Query Builder Basic Mode","path":"/docs/query-your-data/explore-query-data/query-builder/query-builder-basic-mode"},{"title":"Use Advanced Nrql Mode Query Data","path":"/docs/query-your-data/explore-query-data/query-builder/use-advanced-nrql-mode-query-data"},{"title":"Use Advanced Promql Style Mode Query Data","path":"/docs/query-your-data/explore-query-data/query-builder/use-advanced-promql-style-mode-query-data"}]},{"title":"Use Charts","path":"/docs/query-your-data/explore-query-data/use-charts","pages":[{"title":"Chart Refresh Rates","path":"/docs/query-your-data/explore-query-data/use-charts/chart-refresh-rates"},{"title":"Chart Types","path":"/docs/query-your-data/explore-query-data/use-charts/chart-types"},{"title":"Use Your Charts","path":"/docs/query-your-data/explore-query-data/use-charts/use-your-charts"}]}]},{"title":"Nrql New Relic Query Language","path":"/docs/query-your-data/nrql-new-relic-query-language","pages":[{"title":"Get Started","path":"
```

Comment:
I suspect that we didn't intend to include this in the PR? If so, perhaps we could name it something other than "thingystuff.txt" 😂

Reply:
OMG how dare you, `thingystuff` is obviously the backbone of this entire code 😂